### PR TITLE
Minor fixes in the English - grammar, wording, typos

### PR DESCRIPTION
**Pull request by agnesLeroy**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x ] The commit message follows our guidelines

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Correction of typos, suggestion of modification of one sentence in the home page.

### CODE REVIEW

File: _data/home.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 cards:
   - title: "Grid modeling"
     icon: "fas fa-project-diagram"
-    description: "Powsybl provides a complete grid components modeling, which is fully editable and extendable."
+    description: "Powsybl provides a thourough framework for grid components modeling, which is fully editable and extendable."
```
Comment (annetill) [inline _data/home.yml]: I prefer the former formula.

File: _data/home.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ cards:
       url: "/docs/user-stories"
   - title: "Contribute"
     icon: "fas fa-hands-helping"
-    description: "Powsybl is open source and part of the <a href='https://www.lfenergy.org/'>Linux Fundation Energy</a>. Help us to build the next generation power system tool!"
```
Comment (annetill) [inline _data/home.yml]: Are you sure ?

File: _layouts/home.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
                 <h1 class="text-uppercase">powsybl</h1>
                 <h2>Power System Blocks</h2>
                 <img src="/assets/images/powsybl.svg" height="100px" width="100px" class="my-4"/><br/>
-                A open source Java library to assemble applications<br/>for electrical power systems' simulation and analysis
```
Comment (annetill) [inline _layouts/home.html]: Thanks !

File: docs/getting-started.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Powsybl is part of the [LF Energy Foundation](http://www.lfenergy.org), a projec
 
 Powsybl provides an internal complete grid model with substations, voltage levels, AC and DC lines,
 two and three windings transformers, batteries, generators, loads, shunt and static VAR compensators, etc. The grid model can be
-extended with extensions to complete the modelling of the equipments (dynamic profile, short-circuit profile, monitoring, etc.). It also provides importers and exporters for several common pan-european exchange formats ([Entso-E CIM/CGMES](https://www.entsoe.eu/digital/common-information-model/cim-for-grid-models-exchange/), [UCTE-DEF](https://cimug.ucaiug.org/Groups/Model%20Exchange/UCTE-format.pdf), etc.).
+enhanced with extensions that complete the equipments modelling (dynamic profile, short-circuit profile, monitoring, etc.). It also provides importers and exporters for several common pan-european exchange formats ([Entso-E CIM/CGMES](https://www.entsoe.eu/digital/common-information-model/cim-for-grid-models-exchange/), [UCTE-DEF](https://cimug.ucaiug.org/Groups/Model%20Exchange/UCTE-format.pdf), etc.).
```
Comment (annetill) [inline docs/getting-started.md]: Modeling ?

File: docs/user-stories.md [hunk 5]
```diff
@@ -6,6 +6,6 @@ category: User stories
 We advise you to start with our user stories in order to better understand the possibilities of the framework:
 
 - [How to run a load flow with Powsybl ?](/user_stories/loadflow_tutorial.md)
-- [How to calcultate capacity for a Regional Security Center ?](/user_stories/capacity_calculation_rsc.md)
+- [How to calculate capacity for a Regional Security Center ?](/user_stories/capacity_calculation_rsc.md)
```
Comment (annetill) [inline docs/user-stories.md]: Thanks !

File: docs/getting-started.md (hunk 4 above)
Comment (mathbagu) [reply docs/getting-started.md]: Modeling for US / Modelling for GB. I think we should use Modeling.

Comment (agnesLeroy) [reply docs/getting-started.md]: We should be consistent throughout the website: choosing US English we should also make sure words like analyse are written with a "z" (analyze) for example.

File: _data/home.yml (hunk 1 above)
Comment (agnesLeroy) [reply _data/home.yml]: I changed framework into library in the next commit.

File: _data/home.yml (hunk 2 above)
Comment (agnesLeroy) [reply _data/home.yml]: The "to" is actually optional with help, and mostly not used (https://ell.stackexchange.com/questions/47221/help-to-do-something-or-help-do-something).

**Pull request merged**